lua/rest-nvim/utils/init.lua

```diff
@@ -48,27 +48,65 @@ end
 -- variables
 M.read_env_file = function()
   local variables = {}
-  local env_file = "/" .. (config.get("env_file") or ".env")
-
-  -- Directories to search for env files
-  local env_file_paths = {
-    -- current working directory
-    vim.fn.getcwd() .. env_file,
-    -- directory of the currently opened file
-    vim.fn.expand("%:p:h") .. env_file,
-  }
-
-  -- If there's an env file in the current working dir
-  for _, env_file_path in ipairs(env_file_paths) do
-    if M.file_exists(env_file_path) then
-      for line in io.lines(env_file_path) do
-        local vars = M.split(line, "=", 1)
-        variables[vars[1]] = vars[2]
-      end
-    end
-  end
 
-  return variables
+	-- If there is a line at the beginning with @ first
+	if vim.fn.search('^@', 'cn') > 0 then
+		-- Read all lines of the file
+		local lines = vim.api.nvim_buf_get_lines(0, 0, -1, true);
+
+		-- For each line
+		for _, line in pairs(lines) do
+			-- Get all that starts with @
+			if line:match("^@") then
```

Comment:
Use much more concrete regex instead of just `^@`, a much more viable option would be `^@[%w!@#$%^&*-_+?~]+%s*=%s*[^=]+`